stade/core/templates/base.html

```diff
@@ -131,5 +151,41 @@ <h1><a href="{% url 'index' %}"><img src="{% static 'img/isic_logo.svg' %}" alt=
     <div class="help-container">
       <a class="help-button button is-medium is-warning" href="mailto:support@isic-archive.com" target="_blank">Report an Issue</a>
     </div>
+
+    <script type="text/javascript">
```

Comment:
Would it be possible to include this JS and the HTML that it interacts with in a separate template, then `{% include ` it from `base.html`? I think that'll make it easier to maintain, and keep the source of `base.html` from becoming too massive.